pages/api/events/index.ts

```diff
@@ -12,6 +12,18 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
     } = req;
     await prisma.event.create({
       data: {
+        ...(data.opponent && {
```

Comment:
Har du tenkt på tilsvarende for oppdatering av event? For eks hvis stavefeil på motstander ved opprettelse av event